pkg/api/v1/conversion.go

```diff
@@ -43,6 +43,8 @@ func addConversionFuncs() {
 			switch label {
 			case "metadata.name",
 				"metadata.namespace",
+				"metadata.labels",
```

Comment:
Actually, no, my bad, it makes complete sense that you had to add these two.  Carry on....